colcon_ros/task/catkin/build.py

```diff
@@ -8,7 +8,7 @@
 from colcon_core.environment import create_environment_scripts
 from colcon_core.logging import colcon_logger
 from colcon_core.plugin_system import satisfies_version
-from colcon_core.shell import create_environment_hook
+from colcon_core.shell import create_environment_hook, get_shell_extensions
```

Comment:
Please use a separate line for the second import.

Reply:
Fixed